### ISSUE TITLE
test broken after 2.81 release(?)

### ISSUE THREAD

**Issue by julienduroure** [KhronosGroup/glTF-Blender-IO#785]
After commit d0126beb22a72c3be53f9325dc2b181eea2f5664
Seems that there was an error downloading blender 2.81
It was only a few hours after 2.81 release, so maybe relate to new stable version ?
@pjoe

**Comment by scurest** [comment]
Does this fix it? There isn't a 2.81 release on the https://builder.blender.org/download page anymore.

````diff
diff --git a/.circleci/config.yml b/.circleci/config.yml
index 6261f83..1d729bb 100644
--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -33,8 +33,7 @@ jobs:
             fi
             if [[ $FILTER =~ blender28 ]]; then
               echo "Installing Blender 2.81"
-              BLENDER281_URL="https://builder.blender.org$(curl -s https://builder.blender.org/download/ | \
-                grep -oe '[^\"]*blender-2\.81[^\"]*linux[^\"]*-x86_64[^\"]*')"
+              BLENDER281_URL=https://download.blender.org/release/Blender2.81/blender-2.81-linux-glibc217-x86_64.tar.bz2
               mkdir /opt/blender281
               echo "Downloading from: $BLENDER281_URL"
               curl -SL "$BLENDER281_URL" | \
````

**Comment by julienduroure** [comment]
No, we have to switch from 2.81 to 2.82 if there is a 2.82 build already.
Aims is to test with latest dev build, so 2.82 now that 2.81 is official stable
